chainer/links/connection/convolution_3d.py

```diff
@@ -0,0 +1,53 @@
+import numpy
+
+from chainer.functions.connection import convolution_3d
+from chainer import link
+
+
+class Convolution3D(link.Link):
+
+    """Three-dimensional convolutional layer.
+    """
+    def __init__(self, in_channels, out_channels, ksize, stride=1, pad=0,
+                 wscale=1, bias=0, nobias=False, use_cudnn=True,
+                 initialW=None, initial_bias=None):
+        kh, kw, kd = _triplet(ksize)
+        self.stride = _triplet(stride)
+        self.pad = _triplet(pad)
+        self.use_cudnn = use_cudnn
+
+        W_shape = (out_channels, in_channels, kh, kw, kd)
+        super(Convolution3D, self).__init__(W=W_shape)
+
+        if initialW is not None:
```

Comment:
Please use initializers to initialize the weight matrix and the bias vector.
I think the implementation of `Convolution2D` will be helpful.
https://github.com/pfnet/chainer/blob/master/chainer/links/connection/convolution_2d.py#L60